data/brands/shop/storage_rental.json

```diff
@@ -17,6 +17,17 @@
         "shop": "storage_rental"
       }
     },
+    {
+      "displayName": "ALLSAFE Mini Opslag",
```

Comment:
isn't mini Opslag more of a slogan and not part of the name ?

Reply:
To be honest they do use it as title for their website and their listings on Google Maps, but most people would refer to it simply as Allsafe. As a slogan they tend to use `Geeft ruimte`/`Provides space`. Based on that I'd think it's an appropriate display name.